src/StatisticsItem.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Drupal\sapi;
+
+/**
+ * Class StatisticsItem.
+ *
+ * @package Drupal\sapi
+ */
+class StatisticsItem implements StatisticsItemInterface {
+
```

Comment:
If `StatisticsItem` implements `StatisticsItemInterface`, it should contain all the public methods that interface declares. Otherwise the code will break.